packages/entities/entities-plugins/src/components/PluginConfigCard.vue

```diff
@@ -156,6 +194,10 @@ const props = defineProps({
     default: '',
     required: false,
   },
+  showIdAsLink: {
+    type: Boolean,
+    default: false,
+  },
```

Comment:
Everything else looks good to me. Could you document this prop [here](https://github.com/Kong/public-ui-components/blob/main/packages/entities/entities-plugins/docs/plugin-config-card.md#props)?